From 53dab118c8a5121c54dcc2c58e9afefa61a02150 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 13 Dec 2022 16:54:32 +0200
Subject: [PATCH] [SW-0] gaudi2: pci: add error prefix

Add error prefix to phy fw update timeout, so user understand
there is a serious problem and would not ignore this print.

Change-Id: Ife7bd72e6b6e91dd83e82bedbc204223de2ca1af
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 2ce57cc..10a2539 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -357,7 +357,7 @@
 	       regmap_readl(mmPCIE_PHY_PHYX_SRAM_EXT_LD_DONE_3))) {
 
 		if (timeout_expired(start, PCIE_EXT_LD_TIMEOUT_US) && usr_notify) {
-			preboot_puts("ext_ld_done is low > 10mSec\n");
+			preboot_puts("ERROR: ext_ld_done is low > 10mSec\n");
 			usr_notify = false;
 		}
 	}
